src/main/java/ch/wisv/areafiftylan/dto/TransferDTO.java

```diff
@@ -0,0 +1,27 @@
+package ch.wisv.areafiftylan.dto;
+
+/**
+ * Created by beer on 7-1-16.
+ */
+public class TransferDTO {
+
+    String ticketKey;
+
+    String goalUsername;
+
+    public String getTicketKey(){
```

Comment:
Replace the getters and setters with Lombok annotations, see the other DTO's how to do this.